### PR TITLE
Completed es6-template-literals exercise - Paxton Rosenberg

### PR DESCRIPTION
**Pull request by PaxtonRosenberg**
Screenshot 2023-11-20 at 10 38 17 AM

### CODE REVIEW

File: es6-template-literals/index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Javascript ES6 - Template Literals</title>
+  </head>
+  <body>
+    <script>
+      const number = 9200;
+      const street = 'Irvine Center Drive';
+      const address = number + ' ' + street;
```
Comment (robgmerrill) [inline es6-template-literals/index.html:13]: It doesn't look like you used string interpolation here.

File: es6-template-literals/index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+      console.log(`LearningFuze: ${address}`);
+
+      const firstName = 'Uzair';
+      const lastName = 'Ashraf';
+      const age = 25;
+
+      const bio =
+        'My name is ' +
```
Comment (robgmerrill) [inline es6-template-literals/index.html:22]: It doesn't look like you used string interpolation here.

File: es6-template-literals/index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        ' and I am ' +
+        age +
+        ' years old.';
+
+      console.log(`Bio: ${bio}`);
+
+      const fruit = 'apple';
+      const item = 'pen';
+
+      const applePen = fruit + item;
```
Comment (robgmerrill) [inline es6-template-literals/index.html:35]: It doesn't look like you used string interpolation here.

**Review by robgmerrill — CHANGES_REQUESTED** [review]
String interpolation should have been used in the variable definitions, not the console.logs :)

**Comment by PaxtonRosenberg** [comment]
Hey Rob - my apologies! Totally misread the directions. I committed the updated code.

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**